resource-loader.ts

```diff
@@ -0,0 +1,26 @@
+///<reference path=".d.ts"/>
+"use strict";
+
+import * as path from "path";
+
+export class ResourceLoader implements IResourceLoader {
+	constructor(private $fs: IFileSystem,
+		private $staticConfig: Config.IStaticConfig) { }
+
+	resolvePath(resourcePath: string): string {
+		return path.join(__dirname, "../../resources", resourcePath);
```

Comment:
this points to a dir, that is not part of the common lib, which makes this code unusable when you want just the common-lib